tests/unit/models/requests/test_requests.py

```diff
@@ -1,10 +1,15 @@
 from unittest import TestCase
 
-from xrpl.models.requests import Fee
+from xrpl.models.requests import Fee, GenericRequest
 
 
 class TestRequest(TestCase):
     def test_to_dict_includes_method_as_string(self):
         tx = Fee()
         value = tx.to_dict()["method"]
         self.assertEqual(type(value), str)
+
+    def test_generic_request_to_dict_sets_command_as_method(self):
```

Comment:
I believe there's a separate file for testing `GenericRequest` - this test should be there

Reply:
@mvadari That separate file `test_generic_request` is for integration tests. I believe my fix falls under unit tests since I'm just testing model initialization. What do you think?

Reply:
That's fair. I ran them as integration tests to make sure they'd be in a rippled-accepted form, but they could in theory be moved to unit tests